api-specs/Gateway-EE/3.5/kong-ee-3.5.yaml

```
@@ -11906,7 +11906,7 @@ paths:
             description: |
                 Retrieve the current log level of a node.
 
-                See the [Nginx Documentation](ttp://nginx.org/en/docs/ngx_core_module.html#error_log) for the list of possible return values.
+                See the [Nginx Documentation](http://nginx.org/en/docs/ngx_core_module.html#error_log) for the list of possible return values.
```

nit: Would be better to use `https` instead of `http`?

```suggestion
                See the [Nginx Documentation](https://nginx.org/en/docs/ngx_core_module.html#error_log) for the list of possible return values.
```